src/flex-grid/__tests__/flex-grid-item.test.js

```diff
@@ -19,16 +19,76 @@ import {LightTheme} from '../../themes/index.js';
 describe('FlexGridItem', () => {
   test('flexGridItemMediaQueryStyle', () => {
     const testProps = [
-      {flexGridColumnCount: 1, flexGridColumnGap: 0, flexGridRowGap: 0},
-      {flexGridColumnCount: 1, flexGridColumnGap: '1px', flexGridRowGap: '2px'},
-      {flexGridColumnCount: 2, flexGridColumnGap: '1px', flexGridRowGap: '2px'},
-      {flexGridColumnCount: 3, flexGridColumnGap: '1px', flexGridRowGap: '2px'},
+      {
+        flexGridColumnCount: 2,
+        flexGridColumnGap: 0,
+        flexGridRowGap: 0,
+        flexGridItemIndex: 0,
```

Comment:
@leyanlo if you had to change the props fed into the component, does it mean that it's a breaking chnge?

Reply:
Shouldn’t be a breaking change. These props are generated by FlexGrid and passed to FlexGridItems automatically, which is why none of the other usages in this codebase needed to be updated.